### PR TITLE
fix: estimate gas price only in mainnet (chain id 1) and improve logs

### PR DESCRIPTION
**Pull request by mariocao**

### CODE REVIEW

File: price_feed_poller.py [hunk 1]
```diff
@@ -26,11 +26,11 @@ def handle_requestUpdate(
...
       if balance == 0:
           raise Exception("Account does not have any funds")
+      print(f"Balance: {balance} wei")
 
       if gas_price is None:
-          gas_price = w3.eth.generateGasPrice()
-
-      print(f"Got {balance} wei")
+            print(f"Estimating gas price from last blocks...")
+        gas_price = w3.eth.generateGasPrice()
```
Comment (tmpolaczyk) [inline price_feed_poller.py:33]: Wtf

File: tomls/mainnet.toml [hunk 2]
```diff
@@ -11,7 +11,7 @@ pricefeeds = [
 [account]
 # The ethereum address which will create the transactions
 # (needs to have a high enough balance)
-address = ""
+address = "0x102cB4Da51a9936ba2297be8C930B4Ca658586DA"
```
Comment (tmpolaczyk) [inline tomls/mainnet.toml:14]: Was this intended? :P

Comment (mariocao) [reply tomls/mainnet.toml:14]: Nope!!! My bad in the last amend!

File: price_feed_poller.py (hunk 1 above)
Comment (tmpolaczyk) [reply price_feed_poller.py:33]: I mean, doesn't python error on wrong indentation?

Comment (mariocao) [reply price_feed_poller.py]: seems not! :) should be fixed now

**Review by tmpolaczyk — APPROVED** [review]

**Pull request merged**